snapcraft/plugins/kbuild.py

```diff
@@ -112,16 +122,61 @@ def __init__(self, name, options, project):
         if logger.isEnabledFor(logging.DEBUG):
             self.make_cmd.append('V=1')
 
+    def unroll_ubuntu_config(self, config_path):
```

Comment:
is unroll how you kernel people call this?
It sounds weird to me, and given the comment you have down there, assemble_ubuntu_config sounds nicer.